manifest_staging/charts/secrets-store-csi-driver/values.yaml

```diff
@@ -152,3 +153,14 @@ rotationPollInterval:
 
 ## Filtered watch nodePublishSecretRef secrets
 filteredWatchSecret: false
+
+## Prevent the CSI driver from being scheduled on virtual-kublet nodes
+affinity: 
```

Comment:
We can nest this field inside linux and windows fields in the `values.yaml`

Reply:
Hi @aramase,

Thank you for your feedback, I've added the changes here: https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/513/commits/b235b2222c3724a39308eb25c3873b652871504d